src/app/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './App.css';
+
+const App: React.FC = () => {
+    return (
+        <div className="App">
+            <header className="App-header">
+                <p>
+                    Edit <code>src/App.tsx</code> and save to reload.
+                </p>
+                <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
+                    Learn React
+                </a>
+            </header>
+        </div>
+    );
+};
+
+export default App;
```

Comment:
I think we must avoid `default` exports, see a lot of good reasons here: https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html